Program.cs

```diff
@@ -1,4 +1,51 @@
-﻿using System;
-Console.WriteLine ("Hello, World!");
+﻿// --------------------------------------------------------------------------------------------
+// Training ~ A training program for new interns at Metamation, Batch - July 2023
+// Copyright (c) Metamation India.                                              
+// ------------------------------------------------------------------------
+// Program.cs
+// Reverse number guessing game(MSD TO LSD)
+// System will display the number you guessed.
+// --------------------------------------------------------------------------------------------
+using System;
 
-   
\ No newline at end of file
+namespace Training {
+   #region Program ------------------------------------------------------------------------------
+   /// <summary>Reverse number guessing game</summary>
+   internal class Program {
+      #region Methods ---------------------------------------------
+      /// <summary>insert the user to think a ramdom number 1 and 100 </summary>
+      static void Main () {
+         Console.WriteLine ("Welcome to the Guessing Game!");
+         Console.WriteLine ("Think of a number between 1 and 100, and I will try to guess it.");
+         int min = 1;
+         int max = 100;
+         int computerGuess;
+         string userinput = "";

```

Comment:
Move the "string userinput="";" declaration to place where you are reading the input from user. i.e. inside the while loop.